.github/workflows/delta.yaml

```
@@ -77,4 +79,4 @@ jobs:
           rm -rf .tmp/*
         env:
           updates: ${{ inputs.updates }}
-          TMPDIR: .tmp
+          TMPDIR: ${{ runner.workspace }}/.tmp
```

actionlint (1.7.8)

property "workspace" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)